src/AppInstallerCLICore/Workflows/ConfigurationFlow.cpp

```diff
@@ -66,19 +66,28 @@ namespace AppInstaller::CLI::Workflow
 
         constexpr std::wstring_view s_Setting_PowerShellGet_ModuleName = L"name";
 
+        struct PredefinedResourceInfo
+        {
+            std::wstring_view UnitType;
+            bool ElevationRequired = false;
+
+            PredefinedResourceInfo(std::wstring_view unitType) : UnitType(unitType) {}
+            PredefinedResourceInfo(std::wstring_view unitType, bool elevationRequired) : UnitType(unitType), ElevationRequired(elevationRequired) {}
+        };
+
         struct PredefinedResource
         {
             // RequiredModule could be empty, meaning no required modules needed.
             std::wstring_view RequiredModule;
 
-            std::vector<std::wstring_view> UnitTypes;
+            std::vector<PredefinedResourceInfo> ResourceInfos;
         };
 
         std::vector<PredefinedResource> PredefinedResourcesForExport()
         {
             return {
-                { {}, { s_UnitType_WinGetUserSettingsFile_DSCv3 } },
-                { L"Microsoft.Windows.Developer", { L"Microsoft.Windows.Developer/DeveloperMode", L"Microsoft.Windows.Developer/EnableDarkMode", L"Microsoft.Windows.Developer/ShowSecondsInClock", L"Microsoft.Windows.Developer/Taskbar", L"Microsoft.Windows.Developer/WindowsExplorer" } },
+                { {}, { { s_UnitType_WinGetUserSettingsFile_DSCv3 } } },
+                { L"Microsoft.Windows.Settings", { { L"Microsoft.Windows.Settings/WindowsSettings", true } } },

```

Comment:
```suggestion
                { L"Microsoft.Windows.Settings", { { L"Microsoft.Windows.Settings/WindowsSettings", false } } },
```
Due to PowerShell/DSC#786